app/models/like.rb

```diff
@@ -1,5 +1,10 @@
 # A like belongs to a user and a post.
 class Like < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
   belongs_to :post
+
+  # A method that updates the likes counter for a post.
+  def update_counter_cache
+    post.update(likesCounter: post.likes.count)
+  end
```

Comment:
- [ ] I think it would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_like_counter` method every time a new like has been saved to `database`, then make the `update_like_counter` method private 🙃 🙃 
The code can be similar to 👇 

```ruby
  after_save :update_like_counter

  private

  def update_post_counter
    author.increment!(:likes_counter)
  end
```